creator-node/src/config.js

```diff
@@ -453,6 +453,12 @@ const config = convict({
     env: 'pinAddCIDs',
     default: ''
   },
+  cidWhitelist: {
+    doc: 'Array of comma separated CIDs to whitelist',
```

Comment:
maybe comment that the whitelist takes precedence over blacklist

Reply:
added